docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -140,27 +140,30 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 
 ## CSharp API
 
-*To use csharp api for openvino execution provider create a custom nuget package.
+To use csharp api for openvino execution provider create a custom nuget package.
 
-#Windows
+1.Windows
```

Comment:
i think you have to leave a space after '1.' to make it appear as a heading (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

Reply:
done